novawallet/Common/Extension/Foundation/NSAttributedString+Helpers.swift

```diff
@@ -17,3 +18,24 @@ extension NSAttributedString {
         }
     }
 }
+
+extension NSAttributedString {
+    func replacingAttachment<TAttachment, TNewAttachment>(
+        mapClosure: (TAttachment) -> TNewAttachment?
+    ) -> NSAttributedString where TNewAttachment: NSTextAttachment {
+        let mutableAttributedString = NSMutableAttributedString(attributedString: self)
+        let range = NSRange(location: 0, length: mutableAttributedString.string.utf16.count)
```

Comment:
I think it should be NSRange(location: 0, length: mutableAttributedString.length)